video_store_videos.py

```diff
@@ -0,0 +1,66 @@
+import requests
+import datetime
+
+class Video: 
+    def __init__(self, url="http://localhost:5000", selected_video=None):
+        self.url = url
+        self.selected_video = selected_video
+    
+    def print_selected(self):
+        if self.selected_video:
+            print(f"Video with id {self.selected_video['id']} is currently selected\n")
+
+    def create_video(self, title="Default Title", release_date="Today's Date", total_inventory=None):
+        query_params = {
+            "title": title,
+            "release_date": release_date,
+            "total_inventory": total_inventory
+        }
+        response = requests.post(self.url+"/videos",json=query_params)
+        return response.json()
+    
+    def list_videos(self): 
+        response = requests.get(self.url+"/videos")
+        return response.json()
+    
+    def get_video(self, title=None, id=None): 
+        for video in self.list_videos(): 
+            if title:
+                if video["title"]==title:
+                    id = video["id"]
+                    self.selected_video = video
+            elif id == video["id"]:
+                self.selected_video = video
+        
+        if self.selected_video == None: 
+            return "Could not find video by that id or title"
+        
+        response = requests.get(self.url+f"/videos/{id}")
+        return response.json()
```

Comment:
You don't verify the response code here before trying to parse the json:

```suggestion
        if response.status_code == 200:
            return response.json()
        else:
            return { "error": "failed to get video!" }
```